macaroonbakery/discharge.py

```diff
@@ -76,18 +81,19 @@ def check_third_party_caveat(self, ctx, info):
         :param ctx (AuthContext)
         :param info (ThirdPartyCaveatInfo) holds the information decoded from
         a third party caveat id
-        :return: If the caveat is valid, it returns a None error and optionally
-         a slice of extra caveats that will be added to the discharge macaroon
+        :return: If the caveat is validi might return optionally
```

Comment:
:return: An array of extra caveats to be added to the discharge macaroon.

(leaving the description of the returned exceptions for the start) ?

Reply:
done.